### ISSUE TITLE
[feature request] Unity dock behavior.

### ISSUE THREAD

**Issue by VandaLHJ** [psifidotos/Latte-Dock#540]
Can you make it useable as unity panel replacement with ability to stretch it over entire screen and with icons that take either top or bottom places, with space between ?

**Comment by psifidotos** [comment]
> Can you make it useable as unity panel replacement with ability to stretch it over entire screen and with icons that take either top or bottom places, with space between ?

this is already supported... this type of applet alignment is called Justify, you can have your applets, tasks whatever you want at top, center, bottom, all three at the same time...

Latte supports also multiply instances, so you can also have a top dock always visible with Justify mode. In that case your left dock and your top dock/panel will cooperate always in perfect placement for each other concerning the top-left corner of your screen...

In master branch (upcoming 0.7 version) we have also included the Meta+No behavior of activating tasks and of course the nice visual of showing the tasks numbers...
In master branch we also support different layouts, one of them is called Unity... It configures all this for your automatic with one click...

**Comment by VandaLHJ** [comment]
Thanks for the tip. It is possible to use Latte dock for just plasmoid clock and disable any app icons on it ?

**Comment by psifidotos** [comment]
Yes... you can remove the Latte plasmoid... in master branch you can remove it from the Tweaks page

and for v0.6 you can load a configuration file that we provide that doesnt contain the latte plasmoid....
